doc/guides/doc-style-guide.md

```diff
@@ -104,10 +104,14 @@ this guide.
 
 See also API documentation structure overview in [doctools README][].
 
+For topics not covered here, refer to the [Microsoft Writing Style Guide][]. For
+issues not covered there, refer to _The Chicago Manual of Style_.
+
+[Javascript type]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#Data_structures_and_types
+[Microsoft Writing Style Guide]: https://docs.microsoft.com/en-us/style-guide/welcome/
+[`remark-preset-lint-node`]: https://github.com/nodejs/remark-preset-lint-node
```

Comment:
I mentioned this to @addaleax and I don't know of a more appropriate situation to bring it up other than right here, but I wish we could use scoped packages for these: `@nodejs/remark-preset-lint-node`.

Reply:
`@nodejs-core/*` maybe, so we differentiate between external packages and packages targeting collaborators?

Reply:
> [&hellip;] differentiate between external packages and packages targeting collaborators

Yes, exactly.

Reply:
@mmarchini, sorry, not *targeting*, but rather collaborating on. Why can't we use `@nodejs`?

Reply:
IIRC it was originally reserved on npm so we could use it for new modules/standard library to go under this scope (https://github.com/nodejs/TSC/issues/389 for _some_ of the discussions around this). IMO if we end up using it for that purpose, mixing with tooling that is used exclusively on Node.js core will be confusing, so scoping to something specific for the internal workflow of the project makes more sense.

Reply:
We have plans for another package-scoped bare specifier: `@nodejs/resolver-conformance-tests`.

Refs: https://github.com/nodejs/modules/issues/472#issuecomment-584994986